packages/fast-usdc/src/fast-usdc-policy.core.js

```diff
@@ -0,0 +1,75 @@
+/** @file core-eval to publish update to Fast USDC feedPolicy */
+
+import { E } from '@endo/far';
+import { fromExternalConfig } from './utils/config-marshal.js';
+import { FeedPolicyShape } from './type-guards.js';
+
+/**
+ * @import {Passable} from '@endo/pass-style'
+ * @import {BootstrapManifest} from '@agoric/vats/src/core/lib-boot.js'
+ * @import {LegibleCapData} from './utils/config-marshal.js'
+ * @import {FeedPolicy} from './types.js'
+ */
+
+const contractName = 'fastUsdc';
+const FEED_POLICY = 'feedPolicy';
+
+/**
+ * XXX copied from fast-usdc.start.js
+ *
+ * @param {ERef<StorageNode>} node
+ * @param {FeedPolicy} policy
+ */
+const publishFeedPolicy = async (node, policy) => {
+  const feedPolicy = E(node).makeChildNode(FEED_POLICY);
+  await E(feedPolicy).setValue(JSON.stringify(policy));
+};
+
+/**
+ * @param {BootstrapPowers &
+ *  { consume: { chainStorage: Promise<StorageNode> }}
+ * } powers
+ * @param {{ options: LegibleCapData<{feedPolicy: FeedPolicy & Passable}> }} config
+ */
+export const updateFastUsdcPolicy = async (
+  { consume: { agoricNames, chainStorage } },
+  config,
+) => {
+  /** @type {Issuer<'nat'>} */
+  const USDCissuer = await E(agoricNames).lookup('issuer', 'USDC');
+  const brands = harden({
+    USDC: await E(USDCissuer).getBrand(),
+  });
+  const { feedPolicy } = fromExternalConfig(
+    config?.options, // just in case config is missing somehow
```

Comment:
shouldn't that be an error?

Reply:
yes, but this way, we get a pattern mismatch diagnostic instead of a "can't get .options property of undefined" error.

It's evidently not worth the bother, though. taking it out.